.github/workflows/pull-request-opened-notify-slack.yml

```diff
@@ -0,0 +1,11 @@
+name: Send a Slack notification when PRs are opened
+on:
+  pull_request:
+    types:
+      - opened
+jobs:
+  send-slack-notification:
+    if: github.event.pull_request.merged == false &&
+      github.event.pull_request.user.login != 'dependabot[bot]'
```

Comment:
might be worth actually listening for dependabot but that's not even in this repo

Reply:
Adjusted :+1: